test/server/routes/oauth.test.ts

```diff
@@ -39,6 +48,7 @@ describe('server/routes/oauth', () => {
       client_id: application.clientId,
       client_secret: application.clientSecret,
       redirect_uri: application.callbackUrl,
+      scope: 'email account',
```

Comment:
Adding this line caused another 503 error, even though it _should_ accept a `scope` parameter, the issue turned out to be a bug in the Model where the `saveAuthorizationCode` and `saveToken` methods received a `scope` property which was one of:
- string[]
- string
- undefined

So I've added code to correctly parse the scopes now.